plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraType.java

```diff
@@ -189,7 +196,9 @@ public static Optional<CassandraType> toCassandraType(DataType dataType)
             case ProtocolConstants.DataType.FLOAT:
                 return Optional.of(CassandraTypes.FLOAT);
             case ProtocolConstants.DataType.INET:
-                return Optional.of(CassandraTypes.INET);
+                return Optional.of(new CassandraType(
```

Comment:
Is the new dependecy towards `typeManager` added because `IpAddressType` is only on `trino-main` (and not `trino-spi`) ?

Reply:
Nevermind, addressed in https://trinodb.slack.com/archives/CP1MUNEUX/p1656779968805339?thread_ts=1656754605.965319&cid=CP1MUNEUX

Reply:
@findinpath Yes, I should have added the link in the PR description (I'll add now), sorry for the confusion.